modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -221,9 +226,44 @@ private String _buildFragmentHTML(
 		return sb.toString();
 	}
 
-	private String _getAPIURL(ObjectEntry fdsEntryObjectEntry) {
-		Map<String, Object> properties = fdsEntryObjectEntry.getProperties();
+	private String _embedURLParameters(String apiUrl, HttpServletRequest httpServletRequest){
```

Comment:
A nit about naming: by URL parameters one can understand the set of values that go after the `?`. In this case we can talk about tokens instead. 

So, method name could be something like `interpolateURL`, then we can use "token" to name the variables inside the method.

Reply:
Good!